src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -23,11 +23,15 @@ type Props = {
   myProf: boolean;
 };
 
+const scheduleOpenKey = 'profile.schedule.isOpen';
 const GordonSchedulePanel = ({ profile, myProf }: Props) => {
   const [loading, setLoading] = useState(true);
   const [allSchedules, setAllSchedules] = useState<Schedule[]>([]);
   const [selectedSchedule, setSelectedSchedule] = useState<Schedule | null>(null);
   const [selectedCourse, setSelectedCourse] = useState<CourseEvent | null>(null);
+  const [isScheduleOpen, setIsScheduleOpen] = useState<boolean>(
+    localStorage.getItem('isScheduleOpen') === 'true' ?? true,
```

Comment:
Rather than typing the string `'isScheduleOpen'` every time you want to get/set local storage, you should instead use the string stored in the variable `scheduleOpenKey`. That way, if you need to change the key in the future, you only have to change the string once, on line 26 where `scheduleOpenKey` is defined, rather than having to find everywhere that `'isScheduleOpen'` was typed and updating that.

```suggestion
    localStorage.getItem(scheduleOpenKey) === 'true' ?? true,
```